cogs/tools.py

```diff
@@ -134,6 +138,62 @@ async def ask_question(self, ctx, conversation_mode="true", *, question: str):
 
         await ctx.send(embed=embed)
 
+    @commands.command()
+    async def charinfo(self, ctx, *, characters: str):
+        """Shows you information about up to 25 characters."""
+        def to_string(c):
+            digit = f'{ord(c):x}'
+            name = unicodedata.name(c, 'Name not found.')
+            return f'`\\U{digit:>08}`: {name} - {c} \N{EM DASH} <http://www.fileformat.info/info/unicode/char/{digit}>'
+
+        msg = '\n'.join(map(to_string, characters))
+        if len(msg) > 2000:
+            return await ctx.send('Output too long to display.')
+        await ctx.send(msg)
+
+    @commands.command()
+    async def charinfo_uni(self, ctx, *, characters):
+        """Shows you information on up to 25 unicode characters."""
+        match = re.match(r"<(a?):(\w+):(\d+)>", characters)
+        if match:
+            embed = discord.Embed(
+                title="Non-Character Detected",
+                description=(
+                    "Only unicode characters can be processed, but a custom Discord emoji "
+                    "was found. Please remove it and try again."
+                )
+            )
+            embed.colour = discord.Colour.red()
+            await ctx.send(embed=embed)
+            return
+
+        if len(characters) > 25:
+            embed = discord.Embed(title=f"Too many characters ({len(characters)}/25)")
+            embed.colour = discord.Colour.red()
+            await ctx.send(embed=embed)
+            return
+
+        def get_info(char: str) -> Tuple[str, str]:
```

Comment:
You've removed the type hints from 155, but not from here, they need to be removed from here too.

Reply:
Please send a suggestion, to do it instantly

Reply:
@janaSunrise  Just change it manually, I'm not sure what that `m` is referring to, I assume it's member, but it's on multiple lines so there is no need to clutter the PR with more suggestions, manual commit should do it for this.